keep/api/core/db.py

```diff
@@ -2634,6 +2634,11 @@ def get_all_presets(tenant_id: str) -> List[Preset]:
         )
     return presets
 
+# 
+def get_db_and_static_presets_dtos(tenant_id: str) -> List[PresetDto]:
```

Comment:
It's confusing that we have "get_all" and "get_db_and_static_presets_dtos" = "get all + smth". Shouldn't it be "get_db_only_presets" and "get_all"? :)